packages/cozy-mespapiers-lib/src/utils/input.spec.js

```diff
@@ -54,10 +54,10 @@ describe('Input Utils', () => {
       ${0}        | ${{ min: 0, max: 0 }}       | ${true}    | ${false}
       ${0}        | ${{ min: null, max: 20 }}   | ${false}   | ${true}
       ${0}        | ${{ min: null, max: 20 }}   | ${true}    | ${false}
-      ${0}        | ${{ min: 20, max: null }}   | ${false}   | ${true}
+      ${0}        | ${{ min: 20, max: null }}   | ${false}   | ${false}
       ${0}        | ${{ min: 20, max: null }}   | ${true}    | ${false}
-      ${0}        | ${{ min: 10, max: 30 }}     | ${false}   | ${true}
-      ${0}        | ${{ min: 30, max: 10 }}     | ${false}   | ${true}
+      ${0}        | ${{ min: 10, max: 30 }}     | ${false}   | ${false}
+      ${0}        | ${{ min: 30, max: 10 }}     | ${false}   | ${false}
```

Comment:
These fields are not required, we should be able to skip them anyway by not entering anything inside them.

Reply:
Approche changée